compile-test/crm/src/ch/ivyteam/crm/OrderUtil.java

```diff
@@ -2,26 +2,57 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
 
 import ch.ivyteam.ivy.environment.Ivy;
+import ch.ivyteam.ivy.security.IRole;
 import crm.Product;
 
-public class OrderUtil {
-
+public class OrderUtil
+{
   public static List<Product> getProducts()
   {
     List<Product> products = new ArrayList<>();
-    products.add(initProduct(Ivy.cms().co("/Project/Products/Table"), Double.parseDouble(Ivy.var().get("table"))));
-    products.add(initProduct(Ivy.cms().co("/Project/Products/Chair"), Double.parseDouble(Ivy.var().get("chair"))));
+    products.add(initProduct(Ivy.cms().co("/Project/Products/Table"), Double.parseDouble(Ivy.var().get("table")), ClearanceLevel.HEAD));
+    products.add(initProduct(Ivy.cms().co("/Project/Products/Chair"), Double.parseDouble(Ivy.var().get("chair")), ClearanceLevel.AGENT));
     return products;
   }
   
-  private static Product initProduct(String name, double price)
+  public static List<Product> getProductsWithClearance()
+  {
+    IRole role = Ivy.session().getSessionUser().getRoles().get(1);
```

Comment:
I don't like this get(1). 
What if a user theoretically has both roles? What if a user has only role Everybody?

Reply:
Found the proper way to check for roles :smiley: 
We could already merge the other branch (the one I want to merge into) that should automatically re-target the merge target for this branch to master. Or we wait until the core is merged. I'm fine with either way, now that we know that the other branch is still there we won't forget it.